python/lsst/analysis/tools/tasks/gatherResourceUsage.py

```diff
@@ -165,7 +175,6 @@ def run(self, **kwargs: Any) -> Struct:
             .sort_values("task"),
             memrun,
         )
-
```

Comment:
You don't have to change, but typically we try to minimize the unnecessary white space changes.